### DISCUSSION TITLE
Storing comments in issues of gitlab repository

### DISCUSSION THREAD

**Discussion by RomainPuyau** [quarto-dev/quarto-cli#8230] [Q&A]
### Description

Is it possible to add comments like [this](https://quarto.org/docs/output-formats/html-basics.html#commenting)  in a quarto and revealjs presentation (or website) with utterances (or other thing) and issues from a gitlab repository ? I didn't find anything on internet neither in the issues. Sorry for my english..My apologize if this question has been already asked or if it is not right the place to be asked

**Comment by mcanouil** [comment]
The tools for GitHub are basically designed for GitHub use.
Quarto is compatible with mostly GitHub at this point but it is not going to stay that way (cannot give a timeline though).

Quarto cannot create third parties such as Utterances/Giscus, so if those (GitHup Apps) do not exist on other Git platform such as GitLab, there is not much that can be done, but if you do find some tools, please let us know as we might be able to help integrate those tools, either as extensions or within Quarto (no promises).

**Reply by RomainPuyau** [reply]
Unfortunately i don't have the skills to make this tools myselft..

I will wait for this evolution and keep searching a way to resolve this. I'll tell you if something new appear.

Thanks for you answer and wish you an happy new year ! 

long life to quarto ;-)

**Reply by cderv** [reply]
You should follow issues in tools : 

* https://github.com/utterance/utterances/issues/516, https://github.com/utterance/utterances/issues/508
* https://github.com/giscus/giscus/issues/1046


Maybe there are other free tools that support Gitlab, but I don't know them.

**Comment by famuvie** [comment]
Hi, researching a GitLab-based solution for a commenting system on a Quarto blog, I've found the following recent project that seems suitable:

https://belev.me/2025/02/20/Introducing-BeBlob-A-GitLab-Powered-Comment-Section-for-Static-Websites/
